ostia-operator/Gopkg.toml

```diff
@@ -18,7 +18,7 @@ required = [
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
   # branch = "master"
-  version = "=v0.0.5"
+  revision = "2653bb5df7c7926b9fbbcd6a3b3c231db839170b"
```

Comment:
Why not pointing to a released version?

Reply:
I see. Because there are no releases :) 
https://github.com/operator-framework/operator-sdk/releases

Reply:
Exactly, but this gets us aligned anyway with the latest sdk. The build command outputs different manifests on master than on the tag we currently use and we would like to use the latest manifest